setup.py

```diff
@@ -67,19 +67,19 @@
 
                    'License :: OSI Approved :: BSD License',
 
-                   'Programming Language :: Python :: 2',
-                   'Programming Language :: Python :: 2.7',
                    'Programming Language :: Python :: 3',
-                   'Programming Language :: Python :: 3.5',
                    'Programming Language :: Python :: 3.6',
+                   'Programming Language :: Python :: 3.7',
+                   'Programming Language :: Python :: 3.8',
+                   'Programming Language :: Python :: 3.9',
                    ],
 
       install_requires=[
           'six',
           'numpy',
           'scipy',
           'pandas',
-          'livvkit==2.1.6',
+          'livvkit==3.0.1',
```

Comment:
You may want to make this a lower bound so that small changes to LIVVkit don't require updating, but could potentially break w/ new releases.